src/components/color-picker/color-picker.component.less

```diff
@@ -55,10 +59,8 @@ ux-color-picker {
         }
 
         &.ux-selected {
-            border: 5px solid fade(@primary, 50%);
-
-            > button {
-                box-shadow: none;
+            > button > .hpe-icon {
+                display: block;
```

Comment:
Could we add `font-weight: 700` here, to increase visibility of the checkmark a bit?

Reply:
Thanks, I will created a ticket now. Added in the `font-weight` style